terraform/aws/analytical-platform-data-production/airflow/kubernetes-service-accounts.tf

```diff
@@ -0,0 +1,9 @@
+resource "kubernetes_service_account" "airflow" {
+  metadata {
+    namespace = kubernetes_namespace.dev_airflow.metadata[0].name
+    name      = "airflow"
+    annotations = {
+      "eks.amazonaws.com/role-arn" = "arn:aws:iam::593291632749:role/airflow_monitoring_dev"
```

Comment:
It is better to use the output of the role for service account module for the annotation rather than hard-coding it.